src/Metadata/Resource/Factory/OperationDefaultsTrait.php

```diff
@@ -42,7 +42,7 @@ trait OperationDefaultsTrait
 
     private function addGlobalDefaults(ApiResource|Operation $operation): ApiResource|Operation
     {
-        $extraProperties = [];
+        $extraProperties = $this->defaults['extra_properties'] ?? [];
```

Comment:
I don't think `?? []` is needed, but it's nitpicking.

Reply:
yes but because we can't be sure no one is using this trait out of our bundle I think its safer like this